ledger/sandbox-classic/BUILD.bazel

```diff
@@ -204,6 +204,7 @@ test_deps = [
     "//ledger/caching",
     "//ledger/participant-integration-api",
     "//ledger/participant-integration-api:participant-integration-api-tests-lib",
+    "//ledger/error",
```

Comment:
Please keep Bazel deps in alphabetical order (I'm not sure why we don't check this during linting).

Reply:
Interesting that I missed this one. Thanks